tasks/attacker.py

```diff
@@ -0,0 +1,18 @@
+import math
+from utils.config import *
+from utils.geometry import *
+from skills import skills_union
+from skills import sGoToPoint
+from skills import sGoToBall
+from skills import sKickToPoint
+
+
+
+
+class TAttacker(object):
+	def __init__(self, arg):
+		super(TAttacker, self).__init__()
+		self.arg = arg
+	def execute(self,state,bot_id,play_type=0):
```

Comment:
as we have discussed, we are not going to write any execute function here, only write `logic` for target point in `class function`, (like, in TAttacker, you write logic for target point and then you call `sGoToBall.execute()` here we have to write only that logic and return target point as function output).